Example/ViewController.swift

```diff
@@ -271,7 +273,19 @@ class ViewController: UIViewController {
     }
     
     private func presentActionsAlertController() {
-        let alertController = UIAlertController(title: "Start Navigation", message: "Select the navigation type", preferredStyle: .actionSheet)
+        var title: String?
+        switch profileIdentifier {
+        case .automobile, .automobileAvoidingTraffic:
+            title = "Start Driving"
+        case .cycling:
+            title = "Start Cycling"
+        case .walking:
+            title = "Start Walking"
+        default:
+            title = "Start Navigation"
+        }
```

Comment:
This adjusts the title of the action sheet, which is nice, but the button text is more noticeable. It seems to be defined in the storyboard:

https://github.com/mapbox/mapbox-navigation-ios/blob/636a47f559094a385805b8b59d77b074ab2968f3/Example/Base.lproj/Main.storyboard#L48

But that shouldn’t stop us from modifying the title of `startButton` at runtime. If we do that, let’s also make these strings localizable by `NSLocalizableString(…)` so we don’t regress that part of the UI.

Reply:
Ah I missed a method that I could use to set. Fixed now.